### ISSUE TITLE
DataviewJS width in "Reader Mode"

### ISSUE THREAD

**Issue by samuelspagl** [kepano/obsidian-minimal#557]
### Is the bug present when using the default Obsidian theme?

Yes

### Is the bug present when snippets and plugins are disabled?

Yes

### Minimal theme version

6.3.2

### Describe the bug

Using DataviewJS the width in "Reading Mode" is not the "normal" max-width but 100% of the screen.

<img width="1272" alt="Screenshot 2023-04-07 at 20 15 11" src="https://user-images.githubusercontent.com/46893256/230657632-80b630fd-6b51-4a64-b5c1-8009faf7961f.png">

Don't know if it is intended or not 😅🙈.

### Debug info

 I tried to look it up in the `main.css` but I'm not sure if it is the correct line, I changed those variables in my local env and it worked. [(🔗 to line)](https://github.com/kepano/obsidian-minimal/blob/8653921ee2023fc9c4bed39a9bd2b46a0cc8707a/src/css/main.css#L2887)
from:
```css
width: 100%
max-width: 100%
```
to 
```css
width: var(--line-width-adaptive)
max-width: var(--max-width)
```

<img width="1272" alt="Screenshot 2023-04-07 at 20 23 06" src="https://user-images.githubusercontent.com/46893256/230658649-72ebda97-1925-464b-a85f-39cdf82493c5.png">

Thanks a lot in advance :)

**Comment by ToppDev** [comment]
can confirm this bug.

this issue is actually the same as #538 but this issue is more general, it happens with any DataviewJS output which utilizes
- `dv.span(...)`
- `dv.el(...)`

These functions are fine though
- `dv.header(...)`
- `dv.paragraph(...)`

**Comment by kepano** [comment]
This should be fixed in 7.0+
